### ISSUE TITLE
Default preferred receive bitrate?

### ISSUE THREAD

**Issue by fragtion** [essej/sonobus#240]
Hello! 

It's possible to choose a preferred receive bitrate once in a session/channel, and even apply to all. Which is fantastic

But I'm on a 25mbps plan and find that I need to do this far too often (every time I join a new channel, or every time someone else joins the same channel afterwards), in order to avoid some hero blasting 24bit PCM UDP my way 

It would be great if a preferred receive rate could be set, which takes preference over the other person's preferred SEND rate, particularly if my preferred RECEIVE rate is lower than their preferred SEND rate. Once in a channel someone could always override it if they really wanted to (for example, if someone else prefers 24bit pcm as their receive but I insist on sending only 64kbps, or what have you) 

Not critically urgent, but definitely a quality of life improvement that could help avoid this frustration for people with slower connections :)

Thanks!

**Comment by essej** [comment]
Agreed, this has been on my list for a while! I was going to present it as a maximum send quality to receive… so it would only kick in if someone tried to send you higher than the limit, and it would automatically step it down.
